README.md

```diff
@@ -32,16 +32,25 @@ On Linux you can add `-u $(id -u)` to `docker run` to preserve file permissions.
 
 ### Pip
 
-Kapitan needs Python 3.7.
+Kapitan needs Python 3.8.
```

Comment:
Could this be reformulated to show kapitan support Python versions 3.8 up to 3.10 ?

Reply:
Good idea

Reply:
In which case here is a suggestion : 
```suggestion
Kapitan supports Python 3.8, 3.9 and 3.10.
```